### PR TITLE
SAK-41727 Improved styling on ConnectionManger

### PR DESCRIPTION
**Pull request by adrianfish**
Also, improved portal profile popup and tool integrations. Made it
all more consistent.

### CODE REVIEW

**Comment by adrianfish** [comment]
Screen Shot 2019-04-26 at 21 18 37
Screen Shot 2019-04-26 at 21 17 15
Screen Shot 2019-04-26 at 19 48 54
Screen Shot 2019-04-26 at 19 46 09
Screen Shot 2019-04-26 at 19 45 21
Screen Shot 2019-04-26 at 19 43 48
Screen Shot 2019-04-26 at 19 42 42

**Comment by adrianfish** [comment]
I'm not fixing those codacy issues - they're stupid.

File: library/src/morpheus-master/js/src/sakai.morpheus.profile.popup.js [hunk 1]
```diff
@@ -16,204 +16,181 @@
...
+   */
+  profile.attachPopups = function (jqArray, options) {
+
+    if (!(jqArray instanceof $)) {
+        console.log('profile.attachPopups takes a jQuery object array, from a selector');
+        return;
+    }
 
-        });
+    console.log("attachPopups");
```
Comment (ottenhoff) [inline library/src/morpheus-master/js/src/sakai.morpheus.profile.popup.js]: console.log?

**Comment by adrianfish** [comment]
@ottenhoff Good catch, thanks!

**Pull request merged**